sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-full-time-feedback-integration.scheduler.e2e-spec.ts

```diff
@@ -385,5 +395,83 @@ describe(
         });
       },
     );
+
+    it("Should generate a notification to the ministry when there are ecert feedback errors that block funding for a disbursement.", async () => {
+      // Arrange
+      const notificationMessageType =
+        NotificationMessageType.ECertFeedbackFileErrorNotification;
+      // Remove all the eCert feedback file error notification records before
+      // the run of the test, so that it doesn't interfere with the test data.
+      await db.notification.delete({
+        notificationMessage: {
+          id: notificationMessageType,
+        },
+      });
+      const application = await saveFakeApplicationDisbursements(
+        db.dataSource,
+        undefined,
+        {
+          offeringIntensity: OfferingIntensity.fullTime,
+          applicationStatus: ApplicationStatus.Completed,
+          firstDisbursementInitialValues: {
+            coeStatus: COEStatus.completed,
+            disbursementScheduleStatus: DisbursementScheduleStatus.Sent,
+            documentNumber: SHARED_DOCUMENT_NUMBER,
+          },
+        },
+      );
+      mockDownloadFiles(sftpClientMock, [FEEDBACK_ERROR_NOTIFICATION_FILE]);
+      // Queued job.
+      const mockedJob = mockBullJob<void>();
+
+      // Act
+      const result = await processor.processFullTimeResponses(mockedJob.job);
+
+      // Assert
+      expect(result.length).toBe(1);
+      expect(result).toContain("Process finalized with success.");
+      expect(
+        mockedJob.containLogMessages([
+          "File contains 1 records.",
+          `Disbursement feedback error created for document number ${SHARED_DOCUMENT_NUMBER} at line 2.`,
+        ]),
+      ).toBe(true);
+      // When all the records are processed successfully, expect the file to be deleted from SFTP.
+      expect(sftpClientMock.delete).toHaveBeenCalled();
+
+      // Assert
+      const notification = await db.notification.findOne({
+        select: {
+          id: true,
+          messagePayload: true,
+          notificationMessage: {
+            id: true,
+            templateId: true,
+            emailContacts: true,
+          },
+          metadata: { documentNumber: true, errorCodes: true },
+        },
+        relations: { notificationMessage: true },
+        where: {
+          notificationMessage: {
+            id: notificationMessageType,
+          },
+          dateSent: IsNull(),
+        },
+      });
+      expect(notification.metadata?.documentNumber).toBe(
+        SHARED_DOCUMENT_NUMBER,
+      );
+      expect(notification.messagePayload).toStrictEqual({
+        email_address: notification.notificationMessage.emailContacts[0],
+        template_id: notification.notificationMessage.templateId,
+        personalisation: {
+          lastName: application.student.user.lastName,
+          givenNames: application.student.user.firstName,
+          applicationNumber: application.applicationNumber,
+          errorCodes: notification.metadata?.errorCodes,
```

Comment:
The error codes are hard-coded and known and, unless there is a need for the metadata to contain the error codes, we should stop saving them, as per previous comments.